indico/modules/events/surveys/forms.py

```diff
@@ -43,6 +43,21 @@ class SurveyForm(IndicoForm):
     submission_limit = IntegerField(_("Capacity"),
                                     [HiddenUnless('limit_submissions'), DataRequired(), NumberRange(min=1)],
                                     description=_("Maximum number of submissions accepted"))
+    notifications_enabled = BooleanField(_('Notifications'), widget=SwitchWidget(),
+                                         description=_('Whether users are to be notified on specific events within the '
+                                                       'survey'))
+    include_participants_registrants = BooleanField(_('Participants/Registrants'), [HiddenUnless('notifications_enabled')],
+                                                    widget=SwitchWidget(), description=_('Include participants/registrants '
+                                                                                         'to the list of addresses '
+                                                                                         'notified about the start of '
+                                                                                         'the survey'))
+    start_notification_emails = EmailListField(_('Start notification recipients'), [HiddenUnless('notifications_enabled')],
+                                               description=_('List of the addresses that will receive an email '
```

Comment:
You mixed up the descriptions for start/submission notifications.

Also, strange english. "Email addresses to notify about the start of the survey" or something would be better.